xarray/tests/test_dataarray.py

```diff
@@ -5038,8 +5137,17 @@ def test_idxmin(self, x, minindex, maxindex, nanindex, use_dask) -> None:
             result7 = ar0.idxmin(dim="x", fill_value=-5j)
         assert_identical(result7, expected7)
 
-    @pytest.mark.parametrize("use_dask", [True, False])
-    def test_idxmax(self, x, minindex, maxindex, nanindex, use_dask) -> None:
+    @pytest.mark.parametrize(
+        "use_dask", [pytest.param(True, id="dask"), pytest.param(False, id="nodask")]
```

Comment:
Could probably do

```python
pytest.param("True", id="dask", marks=requires_dask)
```

and remove

```python
       if use_dask and not has_dask:
            pytest.skip("requires dask")
```

I think that's slightly more efficient than the current solution but let's leave that for another PR.